icons/googlecardboard.svg

```diff
@@ -1 +1 @@
-<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Google Cardboard icon</title><path d="M23.234 4.932a2.959 2.959 0 00-2.18-.977H3.083c-.827 0-1.58.375-2.18.977C.3 5.534 0 6.361 0 7.188v9.624c0 .827.3 1.654.902 2.256a2.814 2.814 0 002.18.977H7.37a3.22 3.22 0 001.654-.45c.451-.301.902-.753 1.128-1.279l1.203-2.33c.15-.302.376-.452.677-.452.3 0 .601.15.676.451l1.203 2.331c.226.526.677.902 1.128 1.203.451.3 1.053.451 1.58.451h4.285c.827 0 1.58-.376 2.18-.977.602-.602.903-1.429.903-2.256v-9.55c.075-.826-.15-1.653-.752-2.255zm-16.617 9.55c-1.354 0-2.406-1.129-2.406-2.482 0-1.353 1.128-2.406 2.481-2.406s2.406 1.128 2.406 2.481c-.075 1.278-1.128 2.406-2.481 2.406zm10.978 0c-1.354 0-2.482-1.053-2.482-2.407 0-1.353 1.053-2.481 2.407-2.481C18.873 9.594 20 10.647 20 12c0 1.353-1.052 2.481-2.405 2.481Z"/></svg>
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Google Cardboard icon</title><path d="M23.2331 4.932a2.9589 2.9589 0 00-2.1799-.9769H3.0829c-.827 0-1.58.375-2.18.977C.3 5.534 0 6.361 0 7.188v9.6236c0 .827.3 1.654.902 2.256a2.8139 2.8139 0 002.1799.9769h4.2878a3.2199 3.2199 0 001.654-.45c.451-.301.902-.753 1.128-1.279l1.2029-2.3299c.15-.302.376-.452.677-.452.3 0 .601.15.676.451l1.2029 2.331c.226.526.677.902 1.128 1.203.451.2999 1.0529.4509 1.5799.4509h4.2848c.827 0 1.58-.376 2.18-.977.6019-.602.9029-1.429.9029-2.2559V7.187c.075-.826-.15-1.653-.752-2.255zM6.6167 14.4818c-1.3539 0-2.4059-1.129-2.4059-2.4819 0-1.353 1.128-2.406 2.481-2.406 1.3529 0 2.4059 1.128 2.4059 2.481-.075 1.278-1.128 2.4059-2.481 2.4059zm10.9776 0c-1.354 0-2.4819-1.053-2.4819-2.407 0-1.3529 1.053-2.4808 2.407-2.4808 1.3529 0 2.4798 1.053 2.4798 2.4059 0 1.353-1.052 2.4809-2.4049 2.4809z"/></svg>
```

Comment:
![image](https://user-images.githubusercontent.com/1830867/103034964-0ea70000-455e-11eb-9627-3936d9f78e74.png)
Looks like there have been some changes to the Cardboard logo. This is compared to the PNG shown on source's homepage. Can you see if it needs regenerating, or if I'm missing an obvious SVG? 😅

Reply:
I've as well found this SVG (favicon): https://arvr.google.com/cardboard/images/favicon/vr/safari-pinned-tab.svg
And this one (click on "More from Google VR" in the header): https://arvr.google.com/cardboard/images/header/vr-home.svg


Overlaying all four logos, they all look different. 😆 (PNG in orange in the background, our current SVG in blue, the favicon SVG in green and the other SVG in red). And none of the SVGS really matches the PNG. 😄 

![image](https://user-images.githubusercontent.com/70264417/103133149-1fb05800-46a9-11eb-8a84-b857d53980d7.png)


I would suggest using either the favicon SVG or the other one from the website. What do you think?

Reply:
I prefer the line-art one myself too, though I'm worried that people may be using the solid variation and not want the line art. You could always take the line art one and try filling it in, see if that matches up to the PNG source?

Reply:
Well, filling it up isn't a problem at all. I would even prefer a filled version as it is closer to the PNG and our current version.

But the line-art ones won't really match the PNG, as you can see at the "nose".

However, I think I'll just select [this one](https://arvr.google.com/cardboard/images/header/vr-home.svg), fill it up and then we're fine.

Reply:
![googlecardboard](https://user-images.githubusercontent.com/70264417/103136391-46c65400-46c0-11eb-8df0-c0ca8941a1d3.png)

Reply:
Looks good to me - but given the variety between different versions, could another @simple-icons/maintainers sign off on this too?